core/src/main/java/nl/nn/adapterframework/jta/SpringTxManagerProxy.java

```diff
@@ -34,12 +34,12 @@
  * @author  Tim van der Leeuw
  * @since   4.8
  */
-public class SpringTxManagerProxy implements PlatformTransactionManager, BeanFactoryAware {
+public class SpringTxManagerProxy implements PlatformTransactionManager, BeanFactoryAware, IThreadConnectableTransactionManager {
 	private static final Logger log = LogUtil.getLogger(SpringTxManagerProxy.class);
 	
 	private @Setter BeanFactory beanFactory;
 	private @Setter @Getter String realTxManagerBeanName;
-	private PlatformTransactionManager realTxManager;
+	private IThreadConnectableTransactionManager<Object,Object> realTxManager;
```

Comment:
Is het mogelijk de types iets specifieker te maken?

Reply:
Nee, dat gaat helaas niet. De classes die het zouden moeten zijn, zijn niet visibile/refereerbaar.